src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball() {
+
+    }
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    @Override
```

Comment:
The access modifier for the class Ball is missing. It is a good practice to define the access level explicitly, even if you want to use the default package-private level.

Reply:
added getNumber and getColor

Reply:
I created a new constructor and I use it.